lua/nvim-tree/actions/copy-paste.lua

```diff
@@ -234,4 +240,10 @@ function M.copy_absolute_path(node)
   return copy_to_clipboard(content)
 end
 
+function M.setup(opts)
+  M.use_sys_clipboard = opts.actions.use_sys_clipboard
+  if M.use_sys_clipboard == nil then
+    M.use_sys_clipboard = true
+  end
```

Comment:
Can you move the default of this inside `nvim-tree.lua` ? 
Once you did, run the `update-default-opts.sh` script and update the help file with this option :)

Reply:
yes for sure!

Reply:
i've added a small `CONTRIBUTING.md` file complementing this comment

Reply:
perfect! Now it should be fine to be merged. I've update everything 👌